modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -71,107 +53,91 @@ You need to select two different sets of channels, one for {es} and the other fo
 You will need an activation key associated with the correct {es} channels.
 For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
 
+ifeval::[{suma-content} == true]
 
-[IMPORTANT]
-====
-For {es} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
-====
+The channels you need for this procedure are:
 
+[[es-channels-wizard]]
+[cols="1,1,1", options="header"]
+.ES Channels - WebUI
+|===
+| OS Version | Base Channel                     | Tools Channel
+| {es} 6     | RHEL Expanded Support 6          | RES6 SUSE-Manager-Tools x86_64
+| {es} 7     | RHEL Expanded Support 7          | RES7 SUSE-Manager-Tools x86_64
+| {es} 8     | RHEL or SLES ES or CentOS 8 Base | RHEL8-Pool for x86_64
```

Comment:
I'm guessing we need all six of them listed here for RHEL6:

![image](https://user-images.githubusercontent.com/3914967/85358075-2cb38980-b556-11ea-8839-4f2381fdea16.png)

All three of these for RHEL7:

![image](https://user-images.githubusercontent.com/3914967/85358120-481e9480-b556-11ea-8433-d940cc132b85.png)

And just this one for RHEL8:

![image](https://user-images.githubusercontent.com/3914967/85358144-566cb080-b556-11ea-92cf-7979684c41c0.png)


??

Reply:
The problem with RedHat is always the same . Everything is "incomplete". Unlike SLES or CentOS/OL, you cannot just click and start to use it. You get only the half of what you need.

For Expanded Support you need to import the main media into some channel and for native RedHat you need to configure some unknown number of Channels from the CDN.

What we here provide is only a structure which should help the customers to add the reset on.
And because of legacy stuff ES6 is extra bad.
I think we should do a teams session with screen sharing to explain what is used when and how.

Reply:
OK, let's do that once we've merged this